services/motion/client.go

```diff
@@ -110,67 +107,42 @@ func (c *client) MoveOnGlobe(
 	motionCfg *MotionConfiguration,
 	extra map[string]interface{},
 ) (bool, error) {
-	ext, err := vprotoutils.StructToStructPb(extra)
```

Comment:
All the go types -> proto data munging has been moved into the `(MoveOnGlobeReq) ToPRoto()` method to DRY up the code & make it so we can actually test that logic w/o needing to mock the client.